### ISSUE TITLE
MultiObjective's behavior doesn't match its logging

### ISSUE THREAD

**Issue by mrcslws** [facebook/Ax#290]
I found this bug through code inspection, it didn't actually impact me. But it's clearly wrong.

The code: 
https://github.com/facebook/Ax/blob/a3315150edd26fea26052d309665c6eab1f06adc/ax/core/objective.py#L84-L95

Note how the logic doesn't match the comment. When `lower_is_better` is None, it will set the weight to 0.0, but the logged warning implies that it sets the weight to 1.0.

**Here's the bug demonstrated.**

~~~python
from ax.core.metric import Metric
from ax.core.objective import MultiObjective

objective = MultiObjective(metrics=[Metric("m1"), Metric("m2")])
print(objective.weights)
~~~

**Actual Result**

~~~text
[WARNING 04-09 08:24:30] ax.core.objective: metric m1 has not set `lower_is_better`. Treating as `False` (Metric should be maximized).
[WARNING 04-09 08:24:30] ax.core.objective: metric m2 has not set `lower_is_better`. Treating as `False` (Metric should be maximized).

[0.0, 0.0]
~~~

**Expected Result**:
~~~text
[WARNING 04-09 08:24:30] ax.core.objective: metric m1 has not set `lower_is_better`. Treating as `False` (Metric should be maximized).
[WARNING 04-09 08:24:30] ax.core.objective: metric m2 has not set `lower_is_better`. Treating as `False` (Metric should be maximized).

[1.0, 1.0]
~~~

Note that if you're using random scalarization, the bug will be hidden by this other bug: https://github.com/facebook/Ax/issues/289

**Comment by 2timesjay** [comment]
Thanks for catching this. We'll get a fix in shortly.

**Comment by 2timesjay** [comment]
Fixed by release 0.1.11
